docs/roadmap.rst

```diff
@@ -1,70 +1,127 @@
 .. include:: links.rst
 
---------------------
-Development road map
---------------------
+Development road-map
+====================
 This road-map serves as a guide for developers as well as a way for us to
 communicate to users and other stake-holders aboout the expectations they should
 have about the current functionality of the software and future developments.
 
-Version 0.3 (Targetted for March 1st, 2020)
--------------------------------------------
-This version should be considered an early alpha of the software, but will
-contain a full pipeline of processing from a raw BIDS dataset to analyzable data.
+If you want to be part of the team developing this road-map, please make sure you
+take a look into our `Contributors Guidelines <https://www.nipreps.org/community/CONTRIBUTING>`__.
+Then, please write one of the developers to be invited to our bi-weekly meetings.
```

Comment:
Should we use the dmriprep channel on Mattermost for this instead? It's shown as a badge on the README and might be easier to find than the emails in `.mailmap`.

Reply:
Yup, that sounds like a good idea.

Reply:
Do you want to propose an edit?